Define colors & normalize

dejvid15 opened the pull request:
Closes #286 

Defined colors & typography, moved all to the bottom of the stylesheet file

darjanin commented on app/public/css/styles.css:
```diff
@@ -1577,3 +1509,73 @@ td {
+
+:root {
+  --pink: #e72076;
+  --darkGrey: #20323f;
```
Use only dashes to split long names.

dejvid15 replied:
Will do

dejvid15 commented on app/public/css/styles.css:
```diff
@@ -1577,3 +1509,73 @@ td {
+  color: var(--darkGrey);
+}
+
+input:focus,
```
Removed

dejvid15 commented on app/public/css/styles.css:
```diff
@@ -1577,3 +1509,73 @@ td {
+  --pink: #e72076;
+  --darkGrey: #20323f;
+  --grey: #606a71;
+  --lightGrey: #e6e6e6;
```
Ok

dejvid15 commented on app/public/css/styles.css:
```diff
@@ -1577,3 +1509,73 @@ td {
+  --blue: #78c9dd;
+  --red: #cf4040;
+  --orange: #f4b253;
+}
```
Will do

dejvid15 commented on app/public/css/styles.css:
```diff
@@ -1577,3 +1509,73 @@ td {
+  padding: 0;
+  font-family: JTMarnie, Arial, Helvetica, sans-serif;
+  height: 100%;
+  color: var(--darkGrey);
```
Will refactor

dejvid15 commented:
@darjanin Rebased buttons and added `light & dark` variations for `brand` colors & used in buttons. Also implemented the rest of your review comments.
